pro_gan_pytorch/CustomLayers.py

```diff
@@ -19,15 +19,16 @@ def __init__(self, c_in, c_out, k_size, stride=1, pad=0, initializer='kaiming',
         :param bias: whether to use bias or not
         """
         super(_equalized_conv2d, self).__init__()
-        self.conv = th.nn.Conv2d(c_in, c_out, k_size, stride, pad, bias=True)
+        self.conv = th.nn.Conv2d(c_in, c_out, k_size, stride, pad, bias=False)
         if initializer == 'kaiming':
             th.nn.init.kaiming_normal_(self.conv.weight, a=th.nn.init.calculate_gain('conv2d'))
         elif initializer == 'xavier':
             th.nn.init.xavier_normal_(self.conv.weight)
 
         self.use_bias = bias
-
-        self.bias = th.nn.Parameter(th.FloatTensor(c_out).fill_(0))
+        
+        if self.use_bias:
```

Comment:
Hmmm... Cool :+1: This should reduce some GPU memory in case you don't need affine convolutions.